planetary-social/nos
# Send flag content

## Issues covered
 [#1492](https://github.com/planetary-social/nos/issues/1492)
Final of [#1489](https://github.com/planetary-social/nos/issues/1489)

## Description
This PR includes the functionality to send a note  flag publicly or privately to Nos in the new moderation flow.

## How to test
1. Build the app
2. Click the side menu
3. Click the Settings on the side menu
4. Scroll down to turn on the toggle for new moderation flow
5. Go back to the Feed screen by clicking the feed tab at the bottom of the screen
6. Click on the 3 dots on one of the notes.
7. Select `Flag this content` option. 
8. Select an option from the categories displayed.
9. Select an option on how to send the flag.
10. Click send.
11. Please confirm that:
    - The send button at the top right is disabled with opacity until the last selection is made.
    - The send button at the top right is enabled with no opacity when the last selection is made.
    - The send button at the top right changes to a done button when the success screen shows.
    - The cancel button at the top left is hidden when the success screen shows.
    - The right content category and "how to send" options are sent.

## Screenshots/Video

https://github.com/user-attachments/assets/6e28cdcc-d9a4-4ea1-8f5e-f48173b51a61
